resources/views/admin/adminUpdate.blade.php

```diff
@@ -4,7 +4,7 @@
     <div class="container">
         <a href="{{route("admin")}}" class="btn btn-primary col-md-2"> <i class="fas fa-arrow-circle-left"></i> Back </a>
             <div class="row justify-content-center">
-                <div class="col-md-8">
+                <div class="col-md-8.1">
```

Comment:
Is the `.1` a typo? I don't thin it is a valid Bootstrap class with the .1 at the end.

Reply:
I'm going to change it to `col-md-10` when I merge, but if you think that's wrong we can always change it back.